app/Controller/PagesController.php

```diff
@@ -12,151 +12,170 @@
+    {
+    }
+    public function display4()
+    {
+    }
+    public function display5()
+    {
+    }
+    public function display6()
+    {
+    }
+    public function display7()
+    {
+    }
+    public function display8()
+    {
+    }
+    public function display9()
+    {
+    }
+    public function display10()
+    {
+    }
+    public function display11()
+    {
+    }
+    public function display12()
+    {
+    }
+    public function display13()
+    {
+    }
+    public function display14()
+    {
+    }
+    public function display15()
+    {
+    }
+    public function display16()
+    {
+    }
+    public function display17()
+    {
+    }
 
-/**
- * Displays a view
- *
- * @return void
- * @throws NotFoundException When the view file could not be found
- *	or MissingViewException in debug mode.
- */
-	public function display() {
-    $eeeeeeeeee;
-    $aaaaaaaaaa;
-    $bbbbbbbbbb;
-    $aaaaaaaaaa;
+    /**
+     * Displays a view.
+     *
+     * @throws NotFoundException When the view file could not be found
+     *                           or MissingViewException in debug mode.
+     */
+    public function display()
+    {
+        $eeeeeeeeee;
+        $aaaaaaaaaa;
+        $bbbbbbbbbb;
+        $aaaaaaaaaa;
 
-    $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc;
-		$path = func_get_args();
+        $cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc;
+        $path = func_get_args();
 
-		$count = count($path);
-		if (!$count) {
-			return $this->redirect('/');
-		}
-		$page = $subpage = $title_for_layout = null;
+        $count = count($path);
+        if (!$count) {
+            return $this->redirect('/');
+        }
+        $page = $subpage = $title_for_layout = null;
 
-		if (!empty($path[0])) {
-			$page = $path[0];
-		}
-		if (!empty($path[1])) {
-			$subpage = $path[1];
-		}
-		if (!empty($path[$count - 1])) {
-			$title_for_layout = Inflector::humanize($path[$count - 1]);
-		}
-		$this->set(compact('page', 'subpage', 'title_for_layout'));
+        if (!empty($path[0])) {
+            $page = $path[0];
+        }
+        if (!empty($path[1])) {
+            $subpage = $path[1];
+        }
+        if (!empty($path[$count - 1])) {
+            $title_for_layout = Inflector::humanize($path[$count - 1]);
+        }
+        $this->set(compact('page', 'subpage', 'title_for_layout'));
 
-		try {
-			$this->render(implode('/', $path));
-		} catch (MissingViewException $e) {
-			if (Configure::read('debug')) {
-				throw $e;
-			}
-			throw new NotFoundException();
-		}
-  }
+        try {
+            $this->render(implode('/', $path));
+        } catch (MissingViewException $e) {
+            if (Configure::read('debug')) {
```

Comment:
__[staging][phpmd]__
* Avoid using static access to class 'Configure' in method 'display'.